tests/e2e/reflect_helper.go

```diff
@@ -16,9 +18,18 @@ import (
 	"github.com/CosmWasm/wasmd/x/wasm/types"
 )
 
-// InstantiateReflectContract store and instantiate a reflect contract instance
+// InstantiateStargateReflectContract stores and instantiates a pre 2.0 reflect contract instance.
+// This instance still expects the old CosmosMsg.Stargate variant instead of the new CosmosMsg.Any.
+func InstantiateStargateReflectContract(t *testing.T, chain *ibctesting.TestChain) sdk.AccAddress {
+	codeID := chain.StoreCodeFile("../../x/wasm/keeper/testdata/reflect.wasm").CodeID
```

Comment:
Should we modify also the name of the file to give a meaningful name? Something like this
```suggestion
	codeID := chain.StoreCodeFile("../../x/wasm/keeper/testdata/stargate_reflect.wasm").CodeID
```

Reply:
Funny, I was thinking something similar while implementing this. Will do!